doc/install/install-ceph-gateway.rst

```diff
@@ -321,7 +321,7 @@ For ``bind``, add a wildcard to the DNS record. For example::
  *       IN      CNAME   @
 
 Restart your DNS server and ping your server with a subdomain to ensure that
-your ``ceph-radosgw`` daemon can process the subdomain requests::
+your DNS configuration works as expected::
```

Comment:
iiuc, the purpose of setting the DNS record is to enable radosgw to process the subdomain request. so the original doc also makes sense.